### PR TITLE
Add OpenInv integration for restoring inventory of offline players

### PR DESCRIPTION
**Pull request by jacky8399**
OpenInv is a plugin to view and edit inventory of offline players. This PR adds support for restoring ender chest and main inventory of offline players if OpenInv is present.

The test build corrupted the armor content of all previous restores on my test server when upgraded from [the older beta version](https://github.com/danjono/Inventory-Rollback/releases/tag/v2.0-beta), so use with caution.
[test build 2.0-b2.zip](https://github.com/danjono/Inventory-Rollback/files/5934845/test.build.2.0-b2.zip)

### CODE REVIEW

File: src/main/java/me/danjono/inventoryrollback/commands/Commands.java [hunk 1]
```diff
@@ -26,7 +22,7 @@
 import me.danjono.inventoryrollback.gui.menu.PlayerMenu;
 import me.danjono.inventoryrollback.inventory.SaveInventory;
 
-public class Commands extends ConfigData implements CommandExecutor, TabCompleter {
```
Comment (Aurelien30000) [inline src/main/java/me/danjono/inventoryrollback/commands/Commands.java:29]: Are u sure removing that is correct 😅

**Review by Aurelien30000 — APPROVED** [review]
Nice otherwise!

File: src/main/java/me/danjono/inventoryrollback/commands/Commands.java (hunk 1 above)
Comment (jacky8399) [reply src/main/java/me/danjono/inventoryrollback/commands/Commands.java:29]: yes, https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/command/TabExecutor.html

but I am not sure if this interface is there in older versions of Bukkit...

Comment (Aurelien30000) [reply src/main/java/me/danjono/inventoryrollback/commands/Commands.java:29]: Yes, this interface is implemented in 1.8, enough ^^

**Review by Sidpatchy — APPROVED** [review]
Looks good to me, I've put this into use on a few of my servers.

**Pull request closed without merging**